integration-tests/src/test/resources/indexer/wikipedia_parallel_ingest_segment_index_task.json

```diff
@@ -61,7 +61,7 @@
       "forceGuaranteedRollup": "%%FORCE_GUARANTEED_ROLLUP%%",
       "splitHintSpec": {
         "type": "maxSize",
-        "maxSplitSize": 1
```

Comment:
Should we keep one of these tests to use `maxSplitSize` so that we know that the old specs continue to work

Reply:
:+1: Changed `wikipedia_parallel_index_task.json` back to use `maxSplitSize`.